meta-mel-support/recipes-bsp/96boards-tools/96boards-tools_0.7.bb

```diff
@@ -26,7 +27,18 @@ do_install () {
 
     install -d ${D}${sbindir}
     install -m 0755 ${S}/resize-helper ${D}${sbindir}
+
+	install -d ${D}${sysconfdir}/init.d
+	install -m 0755 ${WORKDIR}/resize-helper.sh ${D}${sysconfdir}/init.d/resize-helper.sh
+
+	sed -i -e "s:udevadm settle:$bindir\/udevadm settle:" ${D}${sysconfdir}/init.d/resize-helper.sh
+	sed -i -e "0,/resize-helper/ s:resize-helper:$sbindir\/resize-helper:" ${D}${sysconfdir}/init.d/resize-helper.sh
+	sed -i -e "s:rc3:$sysconfdir\/rc3:" ${D}${sysconfdir}/init.d/resize-helper.sh
+	sed -i -e "s:rc5:$sysconfdir\/rc5:" ${D}${sysconfdir}/init.d/resize-helper.sh
```

Comment:
What exactly is the reason for using $sysconfdir here, rather than ${sysconfdir}? Relying on the target path being exported is pretty pointless. Beyond that, doing search/replace for a word without being more explicit is not a good habit to develop. Either use placeholders (i.e. `@sbindir@/resize-helper` in the file, which you replace with `${sbindir}/resize-helper`) or leave the full paths hardcoded and replace the full paths (i.e. `/usr/sbin/resize-helper` in the file, which you replace with `${sbindir}/resize-helper`.